notes-webapp/src/main/webapp/skin/less/notes/notes.less

```
@@ -60,18 +61,20 @@
     z-index: 1031;
     background: #eeeeee;
 
+    .notes-content-wrapper {
+      min-height: calc(~"100vh - 210px");
+    }
+
     .notes-content {
-      height: calc(~"100vh - 120px");
+      height: calc(~"100vh - 100px");
```

same, even if we shouldn't use the viewport multiple times in the same page, but define it in the parent container and then use flexbox to adjust using `flex-grow-1`, `flex-grow-0`, `flex-column`...

```suggestion
      height: ~"calc(var(--100vh, 100vh) - 100px)";
```